tests/helpers/testers.py

```diff
@@ -382,7 +407,6 @@ def compute(self):
 
 
 class DummyMetricSum(DummyMetric):
-
```

Comment:
pls use yapf

Reply:
Done in https://github.com/PyTorchLightning/metrics/pull/137/commits/f573686899b0adb0dad5b4f253234e25f97c0e26
(sorry It was my understanding that black was used as a code formatting tool for the project)

Reply:
Also, just noted  that the contribution guidelines in `metrics` link to the ones in `pytorch-lightning`, which mention using `pre-commit` but there is no `.pre-commit-config.yaml` in the metrics repository (whilst there is one in the pl repo), should that be added?

Reply:
yes, that would be another nice addition, mind add it in separate PR? :]
cc: @PyTorchLightning/core-metrics

Reply:
https://github.com/PyTorchLightning/metrics/pull/145